subworkflows/nf-core/vcf_qc.nf

```diff
@@ -12,7 +12,7 @@ workflow VCF_QC {
 
     ch_versions = Channel.empty()
 
-    BCFTOOLS_STATS(vcf)
+    BCFTOOLS_STATS(vcf, target_bed)
```

Comment:
To make this work you would need the target_bed files and their tbi indices. So what you can do it pass along an empty `target_bed` and then it should run through.
```suggestion
    BCFTOOLS_STATS(vcf, [])
```